wallet/src/de/schildbach/wallet/ui/dashpay/work/SendInviteWorker.kt

```diff
@@ -122,6 +122,9 @@ class SendInviteWorker @AssistedInject constructor(
 
     private fun createDynamicLink(dashPayProfile: DashPayProfile, cftx: CreditFundingTransaction, aesKeyParameter: KeyParameter): DynamicLink {
         log.info("creating dynamic link for invitation")
+        // dashj Context does not work with coroutines well, so we need to call Context.propogate
+        // in each suspend method that uses the dashj Context
+        org.bitcoinj.core.Context.propagate(wallet.context)
```

Comment:
The application context is injected in the constructor with Hilt already, maybe you can use that instead?

Reply:
Alternatively, you can inject the entire wallet object. It will require providing it in a module, something like this:
```
@Module
@InstallIn(SingletonComponent::class)
object ConfigurationModule {
    @Provides
    fun provideWallet(): WalletApplication = WalletApplication.getInstance()

    @Provides
    fun provideConfiguration(@ApplicationContext context: Context): Configuration =
        Configuration(PreferenceManager.getDefaultSharedPreferences(context), context.resources)
}
```

Reply:
> The application context is injected in the constructor with Hilt already, maybe you can use that instead?

The Context here is the DashJ context object.  While the code uses `wallet.context` to obtain this context, it can also be obtained with `Context.get()`

For now I would like to leave this PR the way it is and consider using DI starting with the above example as part of PR #803 or another task.